From eae5335a5a7e1187d6572cd95b10a59020cc16b3 Mon Sep 17 00:00:00 2001
From: Praveen Chavan <pchavan@codeaurora.org>
Date: Thu, 6 Oct 2016 17:19:17 -0700
Subject: [PATCH] Revert "mediacodec: minijail: Enable exporting of policy
 files"

Nth-time-revert to disable the minijail till policy files are merged
for msmcobalt

This reverts commit 0571b3805d5cdf66d44f02871490a06438161f1a.

Change-Id: Ic125ac15e6a5b8684c40a937b05a6e20b608b522
---
 services/mediacodec/minijail/Android.mk     | 2 +-
 services/mediaextractor/minijail/Android.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/mediacodec/minijail/Android.mk b/services/mediacodec/minijail/Android.mk
index d2becb4..5522ce6 100644
--- a/services/mediacodec/minijail/Android.mk
+++ b/services/mediacodec/minijail/Android.mk
@@ -19,7 +19,7 @@ ifneq (,$(wildcard $(BOARD_SECCOMP_POLICY)/mediacodec-seccomp.policy))
     LOCAL_SRC_FILES += $(BOARD_SECCOMP_POLICY)/mediacodec-seccomp.policy
 endif

-include $(BUILD_SYSTEM)/base_rules.mk
+#include $(BUILD_SYSTEM)/base_rules.mk

 $(LOCAL_BUILT_MODULE): $(LOCAL_SRC_FILES)
 	@mkdir -p $(dir $@)
diff --git a/services/mediaextractor/minijail/Android.mk b/services/mediaextractor/minijail/Android.mk
index 79c5505..da4d631 100644
--- a/services/mediaextractor/minijail/Android.mk
+++ b/services/mediaextractor/minijail/Android.mk
@@ -19,7 +19,7 @@ ifneq (,$(wildcard $(BOARD_SECCOMP_POLICY)/mediaextractor-seccomp.policy))
     LOCAL_SRC_FILES += $(BOARD_SECCOMP_POLICY)/mediaextractor-seccomp.policy
 endif

-include $(BUILD_SYSTEM)/base_rules.mk
+#include $(BUILD_SYSTEM)/base_rules.mk

 $(LOCAL_BUILT_MODULE): $(LOCAL_SRC_FILES)
 	@mkdir -p $(dir $@)
